_posts/2019-08-28-mentor-spotlight-katharine-hyatt.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Mentor Spotlight: Katharine Hyatt"
+date: 2019-08-31 17:00:00
+categories: spotlights
+tags: HackIllinois2019, mentors, featured
+image:
+author: Divey Anand
+---
+
+In our first mentor spotlight this year, we talk with Katharine Hyatt, who mentored at HackIllinois 2019. (add fluff)
```

Comment:
For your "add fluff" you could summarize the interview (specifically parts that talk about her background or how she got involved with open source).

Reply:
Do you think it's better to add something like: 

"In this interview, Katie recounts some of her experiences with open source and gives valuable insight into the community and its growth."

instead of just summarizing what is to be said below?

Reply:
Sounds great!